app/mixins/event-wizard.js

```diff
@@ -41,23 +42,20 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
   },
 
   /**
-   * Save event & related data
-   *
+   * Save event & related data 
+   * 
    * @return {Promise<*>}
    */
   async saveEventData(propsToSave = []) {
     const { event } = this.model;
     const data = {};
-    const results = await Promise.allSettled(propsToSave.map(property => {
-      try {
-        return event.get(property);
-      } catch (e) {
-        if (!(e.errors && e.errors.length && e.errors.length > 0 && e.errors[0].status === 404)) {
-          // Lets just ignore any 404s that might occur. And throw the rest for the caller fn to catch
+    const results = await RSVP.allSettled(propsToSave).then(property => {
+      return event.get(property);
+    }, (e) => {
+        if (!(e.errors && e.errors.length && e.errors.length > 0 && e.errors[0].status === 404))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5583663)